src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/TestSolution/BicepTestProject/results/Colorization/BicepClassifications.bsl

```diff
@@ -1,15 +1,15 @@
 ﻿[Name: type, Text: "description", Start: (Ln: 1, Ch: 2)]
 [Name: operator, Text: "(", Start: (Ln: 1, Ch: 13)]
-[Name: string, Text: "'The globally unique name of the SignalR resource to create.'", Start: (Ln: 1, Ch: 14)]
+[Name: string;Semantic - string;Syntactic - (TRANSIENT), Text: "'The globally unique name of the SignalR resource to create.'", Start: (Ln: 1, Ch: 14)]
```

Comment:
This is expected. We now have classifications from both textmate and semantic colorization. Once the language server is activated, semantic colors will supersede textmate. Apex colorization service doesn't have support to separate the two. I verified it manually and included all the information from apex colorization service to baseline file.

![image](https://user-images.githubusercontent.com/30270536/186527045-743c6a8e-9634-42d4-a32a-3483cde92153.png)

Reply:
Textmate colorization will look like below. Note missing color for 'string' on ln 10
![image](https://user-images.githubusercontent.com/30270536/186527479-ed5bb6a1-1740-41bf-838e-72e3e668223e.png)